src/index.html

```diff
@@ -15,6 +15,15 @@
   <link rel="apple-touch-icon" href="apple-touch-icon.png">
 </head>
 <body>
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-48182293-6', 'auto');
```

Comment:
@nick-a-kitrum Can you remove the key and use the ENV variable `GOOGLE_ANALYTICS_ID` instead?